packages/pwa/app/utils/url.test.js

```diff
@@ -211,21 +331,101 @@ describe('removeQueryParamsFromPath test', () => {
 })
 
 describe('buildPathWithUrlConfig', () => {
-    test('return a new url with locale value as a part of path', () => {
+    test('return a new url with locale and site a part of path', () => {
+        getUrlConfig.mockImplementation(() => ({
+            locale: 'path',
+            site: 'path',
+            showDefaults: true
+        }))
+        const url = buildPathWithUrlConfig('/women/dresses', {locale: 'en-GB', site: 'uk'})
+        expect(url).toEqual('/uk/en-GB/women/dresses')
+    })
+
+    test('return an expected url with only locale since the showDefaults is off', () => {
```

Comment:
```suggestion
    test('return an expected url with no site, no locale, since the showDefaults is off', () => {
```

Reply:
To be more precise, I've changed it to `return an expected url with no site, no locale for default values when the showDefaults is off`